recipes/libyang/all/conanfile.py

```diff
@@ -0,0 +1,61 @@
+from conan import ConanFile
+from conan.tools.cmake import CMake, CMakeToolchain, cmake_layout, CMakeDeps
+from conan.tools.files import get
+
+required_conan_version = ">=1.53.0"
+
+
+class LibYangConan(ConanFile):
+    name = "libyang"
+    license = "BSD-3-Clause license"
```

Comment:
```suggestion
    license = "BSD-3-Clause"
```

Use the short form listed in SPDX: https://spdx.org/licenses/BSD-3-Clause.html